openpifpaf/show/painters.py

```diff
@@ -18,14 +18,14 @@
 class AnnotationPainter:
     def __init__(self, *,
                  xy_scale=1.0,
-                 keypoint_painter=None,
-                 crowd_painter=None,
-                 detection_painter=None):
-        self.painters = {
-            'Annotation': keypoint_painter or KeypointPainter(xy_scale=xy_scale),
-            'AnnotationCrowd': crowd_painter or CrowdPainter(xy_scale=xy_scale),
-            'AnnotationDet': detection_painter or DetectionPainter(xy_scale=xy_scale),
-        }
+                 painters=None):
+        self.painters = {}
+        for painter_key, painter in PAINTERS.items():
```

Comment:
Be as precise as possible with variable names. Change `painter_key` to `annotation_type`.